src/Stratis.Bitcoin.Features.MemoryPool/MempoolManager.cs

```diff
@@ -97,15 +97,19 @@ public Task<List<uint256>> GetMempoolAsync()
         /// <returns>List of transaction information.</returns>
         public List<TxMempoolInfo> InfoAll()
         {
+            this.logger.LogTrace("()");
             // TODO: DepthAndScoreComparator
 
-            return this.memPool.MapTx.DescendantScore.Select(item => new TxMempoolInfo
+            var infoList = this.memPool.MapTx.DescendantScore.Select(item => new TxMempoolInfo
             {
                 Trx = item.Transaction,
                 Time = item.Time,
                 FeeRate = new FeeRate(item.Fee, (int)item.GetTxSize()),
                 FeeDelta = item.ModifiedFee - item.Fee
             }).ToList();
+
+            this.logger.LogTrace("(-):{0}.{1}={2}", nameof(infoList), nameof(infoList.Count), infoList?.Count);
```

Comment:
```
             this.logger.LogTrace("(-):*.{0}={1}", nameof(infoList.Count), infoList.Count); 
```

This is because we don't care how the variable is named, we know it is the return value. And also we don't need `?` Because it cannot be uninitialised